doc/developer-guides/interrupt-hld.rst

```diff
@@ -0,0 +1,476 @@
+.. _interrupt-hld:
+
+Interrupt Management high-level design
+######################################
+
+
+Overview
+********
+
+This document describes the interrupt management high-level design for
+the ACRN hypervisor.
+
+The ACRN hypervisor implements a simple but fully functional framework
+to manage interrupts and exceptions, as show in
+:numref:`interrupt-modules-overview`. In its native layer, it configures
+the physical PIC, IOAPIC, and LAPIC to support different interrupt
+sources from local timer/IPI to external INTx/MSI. In its virtual guest
+layer, it emulates virtual PIC, virtual IOAPIC and virtual LAPIC, and
+provides full APIs allowing virtual interrupt injection from emulated or
+pass-thru devices.
```

Comment:
Is "pass-thru", spelled as it is, correct? (I personally prefer "pass-through" but I may also be old-fashioned)